nf_core/lint.py

```diff
@@ -241,6 +249,32 @@ def check_ci_config(self):
                     self.failed.append((5, "Continuous integration does not check minimum NF version: '{}'".format(fn)))
 
 
+    def check_readme(self):
+        """ Check the repository README file for errors """
+        logging.debug('Checking the repository README')
+        try:
```

Comment:
We already checked, if it is there, so no try except here necessary imo

Reply:
Yes good point. Ok, then perhaps we should stop the tests after each function call if there are any fails? Then if they’re done in order it will be a nice clean run. The total check will need to be moved to the end too.

Reply:
I think that makes sense, yes!